skbio/metadata/_repr.py

```diff
@@ -126,6 +127,16 @@ def _format_positional_metadata_column(self, key, dtype):
         dtype_fmt = '<dtype: %s>' % str(dtype)
         return self._wrap_text_with_indent(dtype_fmt, key_fmt, 1)
 
+    def _process_interval_metadata(self):
+        if (hasattr(self._obj, "has_interval_metadata") and
```

Comment:
is there a valid situation when the obj would not have the `has_interval_metadata` attr?